pkg/agent/controller/networkpolicy/fqdn.go

```diff
@@ -405,80 +409,102 @@ func (f *fqdnController) deleteRuleSelectedPods(ruleID string) error {
 
 func (f *fqdnController) onDNSResponse(
 	fqdn string,
-	responseIPs map[string]net.IP,
-	lowestTTL uint32,
+	newDNSresponseIPs map[string]ipWithTTL,
```

Comment:
In the camel format, it should be `newDNSResponseIPs`, however, I feel DNS and response is kind of duplicate with the function name, and given you name the struct `ipWithTTL`, I would just name the parameter `newIPWithTTLs`, or `newIPWithTTLMap` to correspond to the variable at L424.